src/main/java/seedu/address/ui/CommandHelpMessage.java

```diff
@@ -0,0 +1,95 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.control.TitledPane;
+import seedu.address.logic.commands.CommandEnum;
+
+public class CommandHelpMessage extends UiPart<TitledPane> {
+    private static final String FXML = "CommandHelpMessage.fxml";
+    private static final String ADD_COMMAND_DESCRIPTION = "Adds a flashcard";
+    private static final String ADD_COMMAND_USAGE = "add e/ENGLISH_PHRASE f/FOREIGN_PHRASE";
+    private static final String ADD_COMMAND_EXAMPLES = "add e/Hello f/你好";
+    private static final String DELETE_COMMAND_DESCRIPTION = "Deletes a flashcard";
+    private static final String DELETE_COMMAND_USAGE = "delete CARD_NUMBER";
+    private static final String DELETE_COMMAND_EXAMPLES = "delete 3";
+    private static final String EDIT_COMMAND_DESCRIPTION = "Edits a flashcard";
+    private static final String EDIT_COMMAND_USAGE = "edit CARD_NUMBER ([e/ENGLISH_PHRASE] [f/FOREIGN_PHRASE])";
+    private static final String EDIT_COMMAND_EXAMPLES = "edit 1 e/Hi f/Hola\nedit 1 e/Hello\nedit 1 f/Guten Morgen";
+    private static final String FLIP_COMMAND_DESCRIPTION = "Flips a flashcard to toggle whether its answer is shown";
+    private static final String FLIP_COMMAND_USAGE = "flip CARD_NUMBER";
+    private static final String FLIP_COMMAND_EXAMPLES = "flip 3";
+    private static final String TEST_COMMAND_DESCRIPTION = "Accepts an answer for a flashcard, then displays whether"
+            + " the answer is correct";
+    private static final String TEST_COMMAND_USAGE = "test CARD_NUMBER a/ANSWER";
+    private static final String TEST_COMMAND_EXAMPLES = "test 1 a/good morning";
+    private static final String IMPORT_COMMAND_DESCRIPTION = "Imports flashcards into LingoGO! from a specified"
+            + " CSV file";
+    private static final String IMPORT_COMMAND_USAGE = "upload CSV_FILE_PATH";
+    private static final String IMPORT_COMMAND_EXAMPLES = "upload ./dictionary.csv";
+    private static final String EXPORT_COMMAND_DESCRIPTION = "Exports flashcards to a CSV file located at"
+            + " <TO BE CONFIRMED>";
+    private static final String EXPORT_COMMAND_USAGE = "export CSV_FILE_PATH";
+    private static final String EXPORT_COMMAND_EXAMPLES = "export ./mycards.csv";
```

Comment:
i feel that this should be placed under the individual command classes. the command classes encompass all information about the commands